src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -624,9 +620,40 @@ const formConfig = {
               }
             }
           }
+        },
+        evidenceSummary: {
+          title: 'Summary of evidence',
+          depends: () => {
+            // TODO: needs one of the types of evidence?, checkbox validation?
```

Comment:
@goldenmeanie What are the conditions to show this evidence summary page?

Reply:
@goldenmeanie has indicated that at least one form of evidence should be required to be selected on the evidenceType page, and that at least one item of each selected kind is also required. Therefore, this screen will always be displayed. And those evidence validations need to be updated.